app/models.py

```diff
@@ -0,0 +1,103 @@
+from flask_sqlalchemy import SQLAlchemy
+from passlib.apps import custom_app_context as pwd_context
+# from config.config import app_config
+from flask import g, url_for
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer, \
+    BadSignature, SignatureExpired
+# import random
+# import string
+from app import app
+
+db = SQLAlchemy(app)
+# SECRET_KEY = ''.join(random.choice(string.ascii_uppercase + string.digits)
+#                      for x in range(32))
```

Comment:
Look at imports that are commented out.
Also please arrange the imports in alphabetical order.